src/ios/ContentSync.m

```diff
@@ -610,13 +611,20 @@ + (NSURLRequest*)canonicalRequestForRequest:(NSURLRequest*)theRequest {
 - (void)startLoading {
     NSData *data = [NSData dataWithContentsOfFile:self.request.URL.path];
 
-    // add the no-cache HEADERs to the request while preserving the existing HEADER values.
-    NSDictionary *headers = [NSDictionary dictionaryWithObjectsAndKeys:
-                             [self.request.allHTTPHeaderFields objectForKey:@"Accept"], @"Accept",
```

Comment:
Was there a reason for only preserving the `Accept` field, rather than the the whole header?